['text':' Deserialize the wrapped RPC if it contains Python UDF','line_number':61,'multiline':False]
['text':' Deserialize wrapped RPC if it contains python call','line_number':72,'multiline':False]
['text':' Need this GIL to guard jit::toPyObj and destruct its returned','line_number':90,'multiline':False]
['text':' py::object','line_number':91,'multiline':False]
['text':' py::error_already_set requires GIL to destruct, take special care.','line_number':96,'multiline':False]
['text':' anonymous namespace','line_number':104,'multiline':False]
['text':' py::error_already_set requires GIL to destruct, take special care.','line_number':118,'multiline':False]
['text':' After sync execution or failed async execution return the value as-is.','line_number':128,'multiline':False]
['text':' Pass a dummy message ID since it will be overwritten anyways.','line_number':262,'multiline':False]
['text':' There are request callback impls in Python, where Python','line_number':264,'multiline':False]
['text':' exceptions could be thrown. For releasing Python exception','line_number':265,'multiline':False]
['text':' py::objects, GIL must be held.','line_number':266,'multiline':False]
['text':' Release ownership on py::objects and also restore','line_number':268,'multiline':False]
['text':' Python Error Indicator.','line_number':269,'multiline':False]
['text':' Clear the Python Error Indicator as we has','line_number':270,'multiline':False]
['text':' recorded the exception in the response message.','line_number':271,'multiline':False]
['text':' Pass a dummy message ID since it will be overwritten anyways.','line_number':274,'multiline':False]
['text':' Run backward (TODO: make this async?).','line_number':296,'multiline':False]
['text':' runAsync() starts in the calling thread, but may return an uncompleted','line_number':313,'multiline':False]
['text':' future (though for non-async code, it will typically be completed).','line_number':314,'multiline':False]
['text':' If it was async, our callback will typically be invoked by the','line_number':315,'multiline':False]
['text':' continuation on an at::launch() thread.','line_number':316,'multiline':False]
['text':' namespace rpc','line_number':340,'multiline':False]
['text':' namespace distributed','line_number':341,'multiline':False]
['text':' namespace torch','line_number':342,'multiline':False]
